resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -565,6 +565,7 @@ class KubernetesSuite extends SparkFunSuite
 
 private[spark] object KubernetesSuite {
   val k8sTestTag = Tag("k8s")
+  val pvTestTag = Tag("persistentVolume")
```

Comment:
Is this new tag documented or mentioned somewhere in this PR?

Reply:
This would be a temporary change to enable Jenkins to use Docker as minikube's driver.
This way Kubernetes integration test would be successful.

More about it:
https://issues.apache.org/jira/browse/SPARK-34738?focusedCommentId=17324165&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17324165

Reply:
Yes, that's exactly what I mean. We had better mention this workaround and limitation (ignoring PVC tests) of this PR in the description for the reviewers.
> This would be a temporary change to enable Jenkins to use Docker as minikube's driver.
This way Kubernetes integration test would be successful.

Reply:
I got it :)